core/worker/src/main/java/com/cognifide/aet/worker/drivers/WebDriverHelper.java

```diff
@@ -27,22 +27,18 @@
  */
 public final class WebDriverHelper {
 
-  public static final String NAME = "name";
-
   public static final String NAME_LABEL = "Name";
 
   public static final String NAME_DESC = "Driver name that will be used by other services like Screenshot Collectors";
 
-  public static final String PATH = "path";
-
-  public static final String PATH_LABEL = "Path";
+  public static final String PATH_LABEL = "Custom path to Firefox binary";
 
-  public static final String PATH_DESC = "Custom path to driver binary";
-
-  public static final String SELENIUM_GRID_URL = "seleniumGridUrl";
+  public static final String LOG_FILE_PATH_LABEL = "Path to firefox error log";
 
   public static final String SELENIUM_GRID_URL_LABEL = "Selenium grid URL";
```

Comment:
It look that those constants are only used in `WebDriverProviderConf`. I believe they should be defined here.
This `WebDriverHelper` look like generic class - so there should be no constants that have `Firefox` or `Chrome` words in it.